bindings/go/src/fdb/fdb.go

```diff
@@ -152,6 +151,24 @@ func APIVersion(version int) error {
 	return nil
 }
 
+// Determines if an API version has already been selected, i.e., if
+// APIVersion or MustAPIVersion have already been called.
+func IsAPIVersionSelected() bool {
```

Comment:
Are these functions intended to be added onto the release-5.1 branch, or did they get picked up with another change?

Reply:
Oh, good catch. No, they weren't intended to be added. Let me remove them, but we should make sure they don't disappear when we merge release-5.1 into release-5.2.

Reply:
Okay, I think the way I did it, it won't cause a problem when we merge, but I guess we'll find out.